src/system_descriptor_data.jl

```diff
@@ -297,6 +297,15 @@ const SYSTEM_CATALOG = [
         raw_data = joinpath(PACKAGE_DIR, "data", "RTS-GMLC-master"),
         build_function = build_modified_RTS_GMLC_RT_sys,
         download_function = download_RTS,
+    ),
+     SystemDescriptor(
+        name = "modified_RTS_GMLC_realization_sys",
```

Comment:
Can this be `RTS_GMLC_realization_sys` instead? Someone else may want to modify it for a different reason in the future.

Reply:
I used this name because it is consistent with the other systems also named `modified_RTS_GMLC_XXXX`

Reply:
Doesn't seem like a meaningful convention.